keyhive_wasm/src/js/keyhive.rs

```diff
@@ -255,14 +255,64 @@ impl JsKeyhive {
     }
 
     #[wasm_bindgen(js_name = getAgent)]
-    pub fn get_agent(&self, id: JsIdentifier) -> Option<JsAgent> {
+    pub fn get_agent(&self, id: &JsIdentifier) -> Option<JsAgent> {
         self.0.get_agent(id.0).map(JsAgent)
     }
 
+    #[wasm_bindgen(js_name = getGroup)]
+    pub fn get_group(&self, id: &JsIdentifier) -> Option<JsGroup> {
+        self.0
+            .get_group(GroupId::from(id.0))
+            .map(|g| JsGroup(g.dupe()))
+    }
+
+    #[wasm_bindgen(js_name = docMemberCapabilities)]
+    pub fn doc_member_capabilities(&self, doc_id: &JsDocumentId) -> Vec<SimpleCapability> {
```

Comment:
This looks like a really helpful function! We could move or some equivalent (maybe SimpleCapability is needed for JS?) up into `keyhive_core`, and then wrap it at the Wasm level.